tests/main/interfaces-personal-files/task.yaml

```diff
@@ -46,9 +46,6 @@ execute: |
     echo "When the interface is connected"
     snap connect test-snapd-sh:personal-files
 
-    echo "And not other interfaces are connected"
```

Comment:
Why this change? I think it's useful to test that without home this snap can still access personal files, or am I missing something?

Reply:
This line was included previously because the test-snapd-sh snap was plugging the home interface and by default it is automatic, so we needed to disconnect manually, but with this change the test-snapd-sh snap is not plugging the home interface anymore so this is not needed, in face the test was failing with this line :)

Reply:
Right, makes sense, thanks.